bin/build

```diff
@@ -4,9 +4,14 @@ set -e
 
 MB_EDITION=${MB_EDITION:=oss}
 
+if [ "$MB_EDITION" != ee ] && [ "$MB_EDITION" != oss ]; then
+    echo "MB_EDITION must be either 'ee' or 'oss'."
+    exit 1
+fi
+
 # Generate the resources/version.properties file
 version() {
-    VERSION_INFO=$(./bin/version $MB_EDITION)
+    VERSION_INFO=$(./bin/version)
```

Comment:
./bin/version doesn't actually use any args so we definitely don't need to pass it.

Reply:
it was intended to take `$MB_EDITION` so it could return either a `0.37.x` or `1.37.x` style version, depending on the value. Looks like that commit was lost somewhere along the way.